.travis.yml

```diff
@@ -2,5 +2,7 @@ sudo: required
 language: node_js
 node_js:
   - "0.10"
-before_install:
- - sudo apt-get update -qq
+after_success:
+ - ./node_modules/.bin/istanbul cover -x "**/spec/**" ./node_modules/.bin/_mocha --report lcovonly -- $(find spec -name '*-spec.js') -R spec --require spec/helper.js
+ - cat ./coverage/lcov.info | node_modules/.bin/coveralls
+
```

Comment:
I debated if we should push that into package.json as a script “npm run coverage” or if it made more sense as a standalone script. It’s different from what Jenkins would use (which uses the xunit-file output), so I opted for a direct execution of the specifics  in .travis.yml